admin/app/views/layouts/solidus_admin/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <%= stylesheet_link_tag "solidus_admin", "data-turbo-track": "reload" %>
+    <%= stylesheet_link_tag "solidus_admin_manifest", "data-turbo-track": "reload" %>
```

Comment:
Usually the manifest is not used directly, but just to tell sprockets what assets it needs to compile and keep track of, see app/assets/config/manifest.js and app/assets/javascripts/application.js in a vanilla rails app).

Open to discussing this in case I'm missing an angle, and/or to fix it in a follow up PR

Reply:
Oh that's right, I got confused because in sprockets parlance the manifest is the one in `config/` with _link_ directives, and not the final file with a require in it. 

Not fundamental, but what would you think about removing the `_manifest` suffix?

Reply:
Makes sense, @elia. I changed the configuration of the assets to follow the conventions used in a host application, but namespacing everything within a `solidus_admin` directory. The main CSS manifest file is `app/assets/solidus_admin/application.css`, and the compiled file is saved to `app/assets/build/solidus_admin/tailwind.css`, to follow [tailwindcss-rails conventions](https://github.com/rails/tailwindcss-rails#configuration).